recipes/raylib/all/test_package/conanfile.py

```diff
@@ -5,6 +5,24 @@
 class TestPackageConan(ConanFile):
     settings = "os", "arch", "compiler", "build_type"
     generators = "cmake", "cmake_find_package_multi"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "build_examples": [True, False],
+        "use_external_glfw": [True, False],
+        "opengl_version":[None, "3.3","2.1","1.1","ES 2.0"]
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "build_examples": False,
+        "use_external_glfw": True,
+        "opengl_version" : None
+    }
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
```

Comment:
```suggestion
```

Do not add these options to test package. It should consume directly from the generated package.